components/suggestion/search_channel_suggestion/index.ts

```diff
@@ -17,8 +19,8 @@ type OwnProps = {
 
 const mapStateToProps = (state: GlobalState, ownProps: OwnProps) => {
     return {
-        teammate: getDirectTeammate(state, ownProps.item.id),
-        currentUser: getCurrentUserId(state),
+        teammate: getDirectTeammate(state, ownProps.item.id) as UserProfile,
+        currentUser: getCurrentUserId(state) as string,
```

Comment:
Are these type assertions necessary? `teammate` should be optional based on the code for `SearchChannelSuggestion`, and I think `getCurrentUserId` always returns a string